src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -0,0 +1,80 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Setter
```

Comment:
wydaje mi się, że jak już ta adnotacja to możemy zrezygnować w ciele klasy z dopisywania setterów

Reply:
Też tak myślałam, ale w kilka razy Intelij nie puścił mi tego bez ręcznie stworzonych setterów, podkreślając, że adnotacje przy getterach wymagają stworzenia setterów... Więc zrobiłam je już wszędzie, żeby znowu w kolejnych miejscach się nie wykrzaczało. W innym wypadku nie robiłabym sobie dodatkowej pracy :)